charts/humio-operator/templates/operator-deployment.yaml

```diff
@@ -34,6 +34,14 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
 {{- end }}
+{{- with .Values.image }}
+      image:
+            {{- toYaml . | nindent 8 }}
+{{- end }}
+{{- with .Values.helperImage }}
+      helperImage:
+            {{- toYaml . | nindent 8 }}
+{{- end }}
```

Comment:
What is the purpose with this? Those fields are not valid for a k8s `Deployment` object.

Reply:
I just realized that and I made chnages so the HelperImage and Image are passed as environment variables from this file.